src/modules/claims/claims.service.ts

```diff
@@ -1574,8 +1571,7 @@ export class ClaimsService {
         errors: [ERROR_MESSAGES.NO_CLAIM_RESOLVED],
       };
     }
-    const credentialIsOffChain = isEIP191Jwt(resolvedCredential);
-    return credentialIsOffChain
+    return isEIP191Jwt(resolvedCredential)
       ? this.verifyRoleEIP191JWT(resolvedCredential)
       : this.verifyVc(
```

Comment:
I would also verify vc, sense we created guard for this

Reply:
This is just a validation to make calls, the credentials are type verified during issuer verification.